packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue

```diff
@@ -1,28 +1,69 @@
 <template>
-    <div
-        :class="$style['c-takeawaypayActivation']"
-        data-test-id="takeawaypayActivation">
-        {{ copy.text }}
+    <div :class="$style['c-takeawaypayActivation']">
+        <card-component
+            is-rounded
+            is-page-content-wrapper
+            card-heading-position="center"
+            data-test-id="takeawaypay-activation-component"
+            :class="$style['c-takeawaypayActivation-card']">
+            <activation-logged-in
+                :login-url="loginUrl"
+                :registration-url="registrationUrl" />
+            <activation-not-logged-in
+                :login-url="loginUrl"
+                :registration-url="registrationUrl" />
+            <activation-successful
+                :home-url="homeUrl" />
+            <activation-failed />
+        </card-component>
     </div>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
+import { VueGlobalisationMixin } from '@justeat/f-globalisation';
+import CardComponent from '@justeat/f-card';
+import '@justeat/f-card/dist/f-card.css';
 import tenantConfigs from '../tenants';
+import ActivationLoggedIn from './ActivationLoggedIn.vue';
+import ActivationNotLoggedIn from './ActivationNotLoggedIn.vue';
+import ActivationSuccessful from './ActivationSuccessful.vue';
+import ActivationFailed from './ActivationFailed.vue';
 
 export default {
+
+    components: {
+        CardComponent,
+        ActivationLoggedIn,
+        ActivationNotLoggedIn,
+        ActivationSuccessful,
+        ActivationFailed
+    },
+
+    mixins: [
+        VueGlobalisationMixin
+    ],
+
     props: {
         locale: {
             type: String,
             default: ''
```

Comment:
Probably a good idea to add a validator that rejects any locale other than UK if we're not localising to anywhere else yet.

Reply:
It seems redundant here, as we are using globalisation mixin. Added it while fixing some tests, but removed it now